### ISSUE TITLE
Add descriptive tooltips to data vis/comparison subheaders

### ISSUE THREAD

**Issue by adamkendis** [hackforla/311-data#580]
# Description
All subheaders should have some descriptive help box similar to what the Filter subheaders have, especially Time to Close and Frequency.

## Action Items

- [ ] Add HoverOverInfo component to subheaders in Data Visualization and Comparison Tool. Include clear description of what each chart represents/how to read it.
